### DISCUSSION TITLE
React

### DISCUSSION THREAD

**Discussion by mattsbennett** [Bibleup/bibleup#61] [General]
Any plans to publish a react component? Or guidance from anyone who has tried to use this in a react project?

**Comment by mattsbennett** [comment]
For now I've just used refs (for both the BibleUp instance, and the element I'm binding it to).

**Comment by Bukunmikuti** [comment]
Using refs would work fine (did that with the Nuxt website also). However, publishing a React/Vue is definitely a great idea. I will look into that.
